### PR TITLE
Russian locale added, added ugettext in admin.py, added .po and .mo t…

### PR DESCRIPTION
**Pull request by vsevolod-skripnik**
…o MANIFEST, added {% trans %} in change_list.html

### CODE REVIEW

File: django_celery_beat/locale/ru/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+msgid "queue"
+msgstr "очередь"
+
+#: models.py:252
+msgid "Queue defined in CELERY_TASK_QUEUES"
+msgstr "Очередь, определенная в CELERY_TASK_QUEUES"
+
+#: models.py:255
+msgid "exchange"
+msgstr "обмен"
```
Comment (KostyaEsmukov) [inline django_celery_beat/locale/ru/LC_MESSAGES/django.po]: This doesn't make any sense IMO. I wouldn't translate this, this is a term <http://docs.celeryproject.org/en/latest/userguide/routing.html#exchanges-queues-and-routing-keys>

The same for the routing key, TBH.

File: django_celery_beat/locale/ru/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+msgstr "Аргументы"
+
+#: admin.py:134
+msgid "Execution Options"
+msgstr "Настройки выполнения"
+
+#: admin.py:156
+#, python-brace-format
+msgid "{0} task{1} {2} successfully enabled"
+msgstr "{0} задание{1} {2} успешно включено"
```
Comment (KostyaEsmukov) [inline django_celery_beat/locale/ru/LC_MESSAGES/django.po]: Please consider using plural forms here.

    msg = ungettext(
        "{0} task was successfully enabled",
        "{0} tasks were successfully enabled",
        rows_updated
    ).format(rows_updated)

.

    msgid "{0} task was successfully enabled"
    msgid_plural "{0} tasks were successfully enabled"
    msgstr[0] "{0} задание было успешно включено"
    msgstr[1] "{0} задания были успешно включены"
    msgstr[2] "{0} заданий были успешно включены"
    msgstr[3] "{0} заданий были успешно включены"

File: django_celery_beat/locale/ru/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+msgid "day of month"
+msgstr "день месяца"
+
+#: models.py:151
+msgid "month of year"
+msgstr "месяц в году"
+
+#: models.py:157 models.py:236
+msgid "crontab"
+msgstr "расписание"
```
Comment (vaniakosmos) [inline django_celery_beat/locale/ru/LC_MESSAGES/django.po]: "Crontab" (aka cron table) doesn't have literal translation to Russian. It should be just "кронтаб".
Also "расписаний" means "schedule", it's related to crontab but not the same thing.

**Comment by liquidpele** [comment]
@vsevolod-skripnik Hi there!  It looks like Russian has since been added here:
https://github.com/celery/django-celery-beat/pull/214

Thus I'm closing your PR.  If you see any problems with the translations though please let us know!

**Pull request closed without merging**